guides/common/modules/proc_creating-a-composite-content-view.adoc

```diff
@@ -5,14 +5,14 @@ Use this procedure to create a composite Content View.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-composite-content-view_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content view*.
 . In the *Name* field, enter a name for the view.
 {ProjectName} automatically completes the *Label* field from the name you enter.
 . In the *Description* field, enter a description of the view.
 . Select the *Composite View?* checkbox to create a Composite Content View.
```

Comment:
According to [BZ#2123134](https://bugzilla.redhat.com/show_bug.cgi?id=2123134), there is no **Composite View?** checkbox.

Reply:
> According to [BZ#2123134](https://bugzilla.redhat.com/show_bug.cgi?id=2123134), there is no **Composite View?** checkbox.

I double checked. the outlined steps work in the UI

Reply:
> According to [BZ#2123134](https://bugzilla.redhat.com/show_bug.cgi?id=2123134), there is no **Composite View?** checkbox.

good catch